<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>340</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>490</width>
    <height>340</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>490</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>451</width>
      <height>110</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="layoutInput">
     <item>
      <layout class="QVBoxLayout" name="layoutLabel">
       <item>
        <widget class="QLabel" name="liquidDensity">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Выберите плотность жидкости - ρ [кг/м³]:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="liquidSpeed">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Скорость жидкости - с [м/c]:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="pipeDiameter">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Диаметр трубы - d [мм]:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="dynamicCoeffLiquid">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Динамический коэффициент вязкости - μ [Па*с*10^-6]:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="LayoutLine">
       <item>
        <widget class="QComboBox" name="lineLiquidDensityComboBox"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineLiquidSpeed"/>
       </item>
       <item>
        <widget class="QLineEdit" name="linePipeDiameter"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineDynamicCoeffLiquid"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="labelReynolds">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>230</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Значение коэффициента Рейнольдса:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelValueReynolds">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupReynolds">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>291</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Коэффициент Рейнольдса больше или равен 2300</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>180</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalRoughness">
      <item>
       <widget class="QLabel" name="labelRoughness">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string> Есть ли шероховатость труб?</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalRoughness">
        <item>
         <widget class="QPushButton" name="buttonRoughnessYes">
          <property name="text">
           <string>Да</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonRoughnessNo">
          <property name="text">
           <string>Нет</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="ledges">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>192</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Средняя высота выступа - [мм]:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineLedges">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>80</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="labelValueFriction">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>260</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFriction">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>230</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Коэффициент сопротивления трения:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonStart">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>120</y>
      <width>105</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Посчитать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCheckData">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>260</y>
      <width>120</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Данные плотностей</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCheckResults">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>200</y>
      <width>120</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Результаты</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCheckGraph">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>230</y>
      <width>120</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Построить график</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>490</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
